locale/fi/radiobutton.json

```diff
@@ -0,0 +1,69 @@
+{
+  "title": "Valintanappi (Radio Button)",
+  "intro": "Valintanappi on lomakekomponentti, jolla voi valita yhden vaihtoehdon useammasta valintanapista koostuvasta valintanappiryhmästä. Käyttäjä voi valita kerrallaan vain yhden vaihtoehdon. Kun käyttäjä tekee uuden valinnan, aiemmin tehty valinta poistuu ja korvautuu uudella valinnalla.",
+  "note.title": "Saavutettavuus ja käytettävyys",
+  "note.items": [
+    "Pyri kuvaaviin, lyhyisiin ja ytimekkäisiin otsake-teksteihin (label).",
```

Comment:
I think just "otsakkeisiin" is better than "otsake-teksteihin". In any case there shouldn't be a hyphen.

Reply:
Sure, need to update this to Confluence as well.

Reply:
Fixed.